infra/build/functions/sync/main.py

```diff
@@ -35,41 +41,155 @@ class GitAuth(ndb.Model):
   access_token = ndb.StringProperty()
 
 
-def sync_projects(projects):
+def create_scheduler(cloud_scheduler_client, project_name, schedule):
+  """Creates schedulers for new projects."""
+  project_id = os.environ.get('GCP_PROJECT')
+  location_id = os.environ.get('FUNCTION_REGION')
+  parent = cloud_scheduler_client.location_path(project_id, location_id)
+  job = {
+      'name': parent + '/jobs/' + project_name + '-scheduler',
+      'pubsub_target': {
+          'topic_name': 'projects/' + project_id + '/topics/request-build',
+          'data': bytes(project_name, 'utf-8')
+      },
+      'schedule': schedule
+  }
+
+  try:
+    cloud_scheduler_client.create_job(parent, job)
+    return True
+
+  except exceptions.GoogleAPICallError as error:
+    print(error.message)
+
+  except exceptions.RetryError as error:
+    print(error.message)
+
+  except ValueError:
+    print("Incorrect parameters passed")
```

Comment:
single quotes. also, when does this happen?

Reply:
In case an incorrectly computed cron schedule format is passed or an incorrect timezone